code/processes/wdb.q

```diff
@@ -7,68 +7,71 @@
+                                                                           /-                                              sort mode process to sort the data
+                                                                           /- 3. sort                      -               the process will wait to get a trigger from it's respective
+                                                                           /-                                              save mode process.  When this is triggered it will sort the
+                                                                           /-                                              data on disk, apply attributes and the trigger a reload on the
+                                                                           /-                                              rdb and hdb processes
+
+writedownmode:@[value;`writedownmode;`default];                            /- the wdb process can periodically write data to disc and sort at EOD in two ways:
+                                                                           /- 1. default                   -       the data is partitioned by [ partitiontype ]
+                                                                           /-                                      at EOD the data will be sorted and given attributes according to sort.csv before being moved to hdb
+                                                                           /- 2. partbyattr                -       the data is partitioned by [ partitiontype ] and the column(s) assigned the parted attributed in sort.csv
+                                                                           /-                                      at EOD the data will be merged from each partiton before being moved to hdb
+
+mergenumrows:@[value;`mergenumrows;100000];                                /-default number of rows for merge process
+mergenumtab:@[value;`mergenumtab;`quote`trade!10000 50000];                /-specify number of rows per table for merge process
+
+hdbtypes:@[value;`hdbtypes;`hdb];                                          /-list of hdb types to look for and call in hdb reload
+rdbtypes:@[value;`rdbtypes;`rdb];                                          /-list of rdb types to look for and call in rdb reload
+gatewaytypes:@[value;`gatewaytypes;`gateway];                              /-list of gateway types to inform at reload
+tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];                  /-list of tickerplant types to try and make a connection to
+tpconnsleepintv:@[value;`tpconnsleepintv;10];                              /-number of seconds between attempts to connect to the tp
+tpcheckcycles:@[value;`tpcheckcycles;0W];                                  /-number of attempts to connect to tp before process is killed
+requiredprocs:@[value;`requiredprocs;value'[`hdbtypes`tickerplanttypes]]; 
```

Comment:
Similar to RDB - you don't need to make this a separate variable. I also don't think you need both these - it should only be dependent on tickerplanttypes.  Original version was only dependent on tickerplant types